snappy/scripts/host_speedup.py

```diff
@@ -0,0 +1,73 @@
+import os
+import sys
+import argparse
+import numpy as np
+import matplotlib.pyplot as plt
+from parse_output_file import get_avg_max_cycles, get_avg_host_runtime 
+
+
+# Get the output file directory path
+parser = argparse.ArgumentParser(description='Create graph of DPU speedup over host')
+requiredArgs = parser.add_argument_group('required arguments')
+requiredArgs.add_argument('PATH', help='directory holding output files to parse')
+
+args = parser.parse_args()
+path = args.PATH
+if not os.path.isdir(path):
+	raise argparse.ArgumentTypeError(f"{path} is not a valid path")
+
+	   # 'test file' : ('num_dpus', 'num_tasklets')
+files = {'terror2': ('1',   '4'), 
```

Comment:
One trick I do is I label the files with the parameters, so one example here might be: "terror2_dpus=4_tasklets=12" and then parse it out using a regex, so that I can generate a bunch of different parameters but they're clearly labeled :)